superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -581,22 +581,22 @@ export default class ResultSet extends React.PureComponent<
     const isAdmin = !!this.props.user?.roles?.Admin;
     const displayMaxRowsReachedMessage = {
       withAdmin: t(
-        `The number of results displayed is limited to %(rows)d by the configuration DISPLAY_MAX_ROWS. `,
+        'The number of results displayed is limited to %(rows)d by the configuration DISPLAY_MAX_ROWS. ',
         { rows },
       ).concat(
         t(
-          `Please add additional limits/filters or download to csv to see more rows up to `,
+          'Please add additional limits/filters or download to csv to see more rows up to ',
         ),
-        t(`the %(limit)d limit.`, { limit }),
+        t('the %(limit)d limit.', { limit }),
```

Comment:
nit: Is there a reason we use this `concat` pattern instead of the method in `App/index` above? is it a jsx vs. tsx thing? My understanding is we could do this here, but correct me if I'm wrong. Is it better to translate smaller segments at a time?
```
.concat(
  t(
    'Please add additional limits/filters or download to csv to see more rows up to ' +
    'the %(limit)d limit.', { limit }
  )
)
```

Reply:
That's a great point, thanks for catching. Let me clean these disconnected segments 👍